calc.js

```diff
@@ -1,294 +1,85 @@
 window.onload = function(){
+    
+    var calculator = document.getElementById("calculator");
+    var keys = calculator.querySelector(".keys");
+    keys.addEventListener('click', function(e){
+        if(e.target.attributes.type.value === "numeral"){
```

Comment:
Look into using a switch statement here. If you have a bunch of if/elseifs switches are better

Reply:
Like this?

    keys.addEventListener('click', function(e){ 
        switch(e.target.attributes.type.value){
            case "numeral":
                digitPressed(e.target.innerHTML);
                console.log("You pressed a number!");
                break;
            case "operator":
                operatorPressed(e.target.innerHTML);
                e.target.classList.add ("pressed");
                console.log("You pressed an operator!");
                break;
            case "negative":
                negativePressed(e.target.innerHTML);
                console.log("You pressed negative!");
                break;
            case "reset":
                clear();
                console.log("Cleared!");
                break;
            case "squared":
                squaredPressed();
                console.log("Squared pressed!");
                break;
            case "square-root":
                squareRootPressed(); 
                console.log("Square Root pressed!");
                break;  
        }});

It worked so I'm assuming "yes" ha!